chart/templates/jobs/mongodb-migration/job.yaml

```diff
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright 2022 The HuggingFace Authors.
 
+{{- if .Values.dockerImage.jobs.mongodbMigration }}
```

Comment:
what is the use case for not applying the migration scripts? my idea was always to check if a migration is needed (the migrations are stored in the database, so we don't apply the same migration twice or if not needed).